.rubocop.yml

```diff
@@ -141,7 +144,9 @@ Style/TernaryParentheses:
 # I am a huge fan of using trailing commas when I break an argument
 # list down one-per line.
 #
-# As such, I reject this test.
+# As such, I reject these tests.
 #
+Style/TrailingCommaInArguments:
```

Comment:
Isn't this cop for a final trailing comma in a method's arguments, per https://rubocop.readthedocs.io/en/latest/cops_style/#styletrailingcommainarguments ?  Which I would want the cop to whistle about.

Reply:
Yes, you are correct about what Style/TrailingCommaInArguments does.

We differ, in that I like the final trailing comma in a multi-line list of arguments.